WcaOnRails/app/assets/javascripts/application.js

```diff
@@ -107,6 +107,15 @@ wca.competitionsToMarkers = function(map, competitions) {
   return markers;
 };
 
+wca.renderMarkdownRequest = function(markdownContent) {
```

Comment:
This is not your fault, and I don't know the original reason, but `application.js` is not intended for JavaScript code! See line 7.